CDCgov/cdc-open-viz
# Feature/dev 9662

## [Replace With Ticket Number]



## Testing Steps
Open Line chart
On Data Series Panel update series key name for something long text
On Visual Panel click checkbox "Append Series Name to End of Line Charts".
Result: If series name does not fit it should be truncated (...)
See attached image
<img width="1462" alt="Screenshot 2024-11-20 at 16 56 15" src="https://github.com/user-attachments/assets/b9e55d42-69ca-4c14-8f7b-2503716988cb">





## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)



## Additional Notes
